libsolidity/codegen/ir/IRGenerator.cpp

```diff
@@ -211,6 +200,7 @@ string IRGenerator::generate(
 	bool creationInvolvesAssembly = m_context.inlineAssemblySeen();
 	t("memoryInitCreation", memoryInit(!creationInvolvesAssembly));
 
+	set<string> usedSourceNames = m_context.usedSourceNames();
 	resetContext(_contract, ExecutionContext::Deployed);
```

Comment:
`resetContext()` should clear `usedSourceNames()`.

Reply:
It does.

Reply:
Ah, right. It just creates a new context :)